requirements.txt

```diff
@@ -3,4 +3,4 @@ toml
 debugpy
 wget
 multiversx-sdk-cli==9.6.2
-multiversx-sdk==0.9.0
\ No newline at end of file
+multiversx-sdk==0.10.0
```

Comment:
You can use even more recent ones, without changes, generally speaking. Micro-breaking changes (between two different versions) are mentioned in the release notes:

https://github.com/multiversx/mx-sdk-py/releases

Reply:
This PR will be used mostly for an upcoming upgrade procedure.
I plan to do a bigger overhaul of the main branch afterwards in which we switch to the latest mxpy libs, integrate abis here and there and also get rid of the deprecated tx factories.